.github/workflows/ci.yml

```
@@ -300,3 +300,42 @@ jobs:
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+
+    - name: Install oasdiff
+      run: go install github.com/oasdiff/oasdiff@latest
+
+    - name: Mark allow-breaking if commit message requests it
+      id: allow_breaking
+      run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:2:38: Double quote to prevent globbing and word splitting

(shellcheck)